nautobot/cloud/templates/cloud/cloudnetwork_retrieve.html

```diff
@@ -9,19 +9,38 @@
 
 
 {% block extra_nav_tabs %}
-    {% if object.parent is None %}
+    {% if children_count %}
             <li role="presentation"{% if request.GET.tab == 'children' %} class="active"{% endif %}>
                 <a href="{{ object.get_absolute_url }}#children" onclick="switch_tab(this.href, reload=false)" aria-controls="children" role="tab" data-toggle="tab">
-                    Children {% badge object.children.count %}
+                    Children {% badge children_count %}
                 </a>
             </li>
-    {% else %}
-        {{ block.super }}
```

Comment:
Why was this removed? Weird things happen if the if statement above evaluates to false.

Reply:
Ah, I see, we can have additional cases below it. I still believe we need a final `block super` in case none of the tabs need to load.

Reply:
IIRC the block.super is just an empty block, so it shouldn't be needed here.

Reply:
Okay. I had an issue where the tabs were still rendering even if they were child objects that couldn't have children. Calling `block.super` resolved it for me.

Reply:
I reproduced it, changing to the count above allows for the removal of this check.  Thanks for confirming Glenn!